app/code/Magebit/PageListWidget/view/frontend/templates/widget/page_list.phtml

```diff
@@ -0,0 +1,23 @@
+<?php
+/** @var \Magebit\PageListWidget\Block\Widget\PageList $block */
+?>
+<div class="magebit-page-list-widget">
+    <?php if ($title = $block->getTitle()): ?>
+        <h2 class="page-list-title"><?= $block->escapeHtml($title) ?></h2>
+    <?php endif; ?>
+
+    <?php $pages = $block->getPages(); ?>
```

Comment:
For more readable code, put the variable deffinitions at the top of the file. Then just use $pages